ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -369,6 +478,18 @@ def _neighbors(self) -> list | None:
                     neighbor = self._create_neighbor(data["ip_address"], route_client, self.shared_utils.bgp_peer_groups["evpn_overlay_peers"]["name"])
                     neighbors.append(neighbor)
 
+            if self.shared_utils.wan_role == "client":
+                if not self._router_id_listen_ranges(self._wan_listen_ranges):
+                    raise AristaAvdError(f"Loopback0 IP {self.shared_utils.router_id} is not in the Route Reflector listen range prefixes.")
```

Comment:
I think this raise should be moved to the function, so we can raise hinting to the route reflector missing the range. Alternatively move the check inside the for loop below so we check them one by one.

Reply:
not all route reflectors may be on all path-groups and so some listen-range missing may not be bad so there is maybe some more logic to deal with here. What is checked is that we can connect to "at least one RR"

Reply:
add variable path to the listen_range in the bgp peer groups